### DISCUSSION TITLE
Downloaded Video Is Encrypted.

### DISCUSSION THREAD

**Discussion by ujjwaldethe** [aclap-dev/video-downloadhelper#1427] [Help and Q&A]
### Your question

I have downloaded a video from paid website but it is totally encrypted.please help me to decrypt the video
![photo_2024-11-02_13-53-38](https://github.com/user-attachments/assets/809263fa-e1f6-4513-9769-2236bd238084)

### Addon & CoApp version and details

I have downloaded a video from paid website but it is totally encrypted.please help me to decrypt the video
![photo_2024-11-02_13-53-38](https://github.com/user-attachments/assets/809263fa-e1f6-4513-9769-2236bd238084)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

It appears that your content is protected by DRM.  Search this forum for discussions of DRM.  The video you downloaded cannot be decrypted.  The decryption is intentional.  Read some of those threads I just told you to search for & you will learn more about this subject.

**Reply by ujjwaldethe** [reply]
I don't understand how to do it .. please contact me on telegram : @private_ksd

**Reply by GrampaWildWilly** [reply]
No.  Absolutely not.  I will not contact anybody except via posts here on GitHub.

You need to read the reference material I told you to read.  You need to put in some effort to help yourself.  That's what this forum is all about.  It's an international self-help forum of mostly users of the software.  You need to use the search fields here on GitHub.  There is a search field at the top of this page & every discussion page of this forum.  There is also a search field at the top of the home page of this forum.  If you don't know how to do it, guess.  It's not that complicated.  Teach yourself by experimentation and trial & error.
